ecosystem/indexer/src/models/transactions.rs

```diff
@@ -327,7 +327,7 @@ impl BlockMetadataTransaction {
             round: *tx.round.inner() as i64,
             // TODO: dummy/bad values for now. Need to decide how to support this
             previous_block_votes: serde_json::to_value(&tx.previous_block_votes).unwrap(),
-            proposer: tx.proposer.inner().to_string(),
+            proposer: serde_json::to_value(&tx.proposer).unwrap().to_string(),
```

Comment:
please change this column to a jsonb type, so we don't need the `.to_string()` :-)